install.sh

```diff
@@ -1,7 +1,69 @@
 #!/bin/bash
 
+#repo addresses
+aasdkRepo="https://github.com/OpenDsh/aasdk"
+gstreamerRepo="git://anongit.freedesktop.org/gstreamer/qt-gstreamer"
+openautoRepo="https://github.com/rhysmorgan134/openauto.git"
```

Comment:
This will change at some point, I trust.

Reply:
For sure, I won't be submitting it until it's pointing back at the main repo, got somethings to test from slack which hopefully will highlight the cause of the issue.